lib/matrix_api_lite/model/event_types.dart

```diff
@@ -96,4 +94,24 @@ abstract class EventTypes {
   // Spaces
   static const String spaceParent = 'm.space.parent';
   static const String spaceChild = 'm.space.child';
+
+  // MatrixRTC
+  static const String GroupCallMember = 'com.famedly.call.member';
```

Comment:
Why are these uppercae, but the others before it are all lower case?

Reply:
hm only spaceChild and spaceParent are lowerCase, do we want to fix that in this breaking release as well?

Reply:
Ah, well, I didn't uncollapse that far, I guess.